chapters/annotations.tex

```diff
@@ -1387,7 +1389,7 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 
 If \lstinline!colorSelector = true!, it indicates that an rgb-value selector can be presented for a vector of three elements and generate values 0..255 (the color selector should be useable both for vectors of \lstinline!Integer! and \lstinline!Real!).
 
-A \fmtannotationindex{Selector} displays a file dialog to select a file.
+Modifying the \fmtannotationindex{Selector} of \lstinline!loadSelector! or \lstinline!saveSelector! displays a file dialog to select a file.
```

Comment:
Yes, this is an improvement.  I'm just not sure about the use of _modifying_ to express what we want in the context of annotations – there are currently no uses of _modifying_ in the chapter.  Perhaps presence of an annotation is a better terminology to use?
```suggestion
Presence of a \fmtannotationindex{Selector} for \lstinline!loadSelector! or \lstinline!saveSelector! displays a file dialog to select a file.
```

Reply:
I'll reformulate.

Reply:
I realized that "display" is misleading since the annotation doesn't directly display the dialog.

Reply:
This looks OK now.  Resolving.